Orange/tests/test_distances.py

```diff
@@ -34,6 +37,157 @@ def test_pickling(self):
         self.assertTrue(tables_equal(unpickled_dist.col_items, self.dist.col_items))
         self.assertEqual(unpickled_dist.axis, self.dist.axis)
 
+    @contextmanager
+    def named_file(self, content, encoding=None):
+        import tempfile
+        import os
+        file = tempfile.NamedTemporaryFile(
+                "wt", delete=False, encoding=encoding)
+        file.write(content)
+        name = file.name
+        file.close()
+        yield name
+        os.remove(name)
```

Comment:
Should we have a contextmanager like this implemented properly and available in some utils module?
We appear to be using something similar in multiple places in Orange (search "NamedTemporaryFile")